config/rbac/role.yaml

```diff
@@ -6,6 +6,13 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
+  - patch
```

Comment:
Why is `create` not sufficient?

Reply:
I dunno. I copied it from the kubebuilder book: https://book-v1.book.kubebuilder.io/beyond_basics/creating_events.html